src/devSVG.cpp

```diff
@@ -103,7 +103,14 @@ inline void write_attr_str(FILE* f, const char* attr, const char* value) {
   fprintf(f, " %s='%s'", attr, value);
 }
 
-inline void write_attrs_linetype(FILE* f, int lty, double lwd, int col) {
+inline void write_attrs_linetype(FILE* f, const pGEcontext gc) {
+  int lty = gc->lty;
```

Comment:
I don't think adding these helper variables adds much to readability, so I'd drop them.